copier/main.py

```diff
@@ -376,13 +376,21 @@ def jinja_env(self) -> SandboxedEnvironment:
         """
         paths = [str(self.template.local_abspath)]
         loader = FileSystemLoader(paths)
+        default_extensions = [
+            "cookiecutter.extensions.JsonifyExtension",
+            "cookiecutter.extensions.RandomStringExtension",
+            "cookiecutter.extensions.SlugifyExtension",
+            "jinja2_time.TimeExtension",
```

Comment:
This is a convenient extension too, but again `AnsibleCoreFiltersExtension` would do the trick with `{{ '%Y-%m-%d %H:%M:%S' | strftime }}`.

So, in this case also, please remove the whole dependency.

I think we can live with just ansible-core-filters shipped by default.

---

😢 Sorry! I know it was me who asked you to include those dependencies... but after seeing the code in place, seeing how ansible-core-filters does the job, and also seeing how easy it would be to add extra extensions, I changed my mind. I'm very sorry! 🙏🏼

Reply:
Hahaha it's totally fine don't worry! :joy: